ci/results.go

```diff
@@ -34,6 +34,7 @@ func ExtractResult(out, secret string, execTime time.Duration) (*Result, error)
 		// check if line has expected JSON score string
 		if score.HasPrefix(line) {
 			sc, err := score.Parse(line, secret)
+			log.Println("ci got score parsing ci results, line: ", sc)
```

Comment:
It seems `ExtractResult()` is only used from `web/hooks.go:runTests()` which also has access to the logger, and so we can also pass that logger to `ExtractResults()` as the first argument. Thus, we can replace these `log.Println` statements with appropriate `logger.Debug` statements.